dips/dip-158.md

```diff
@@ -0,0 +1,795 @@
+---
+dip: 158
+title: P2M (eCommerce) Payments Off-chain API
+authors: Daniel Prinz, Yinon Zohar, Kevin Hurley (@kphfb)
+status: Draft
+type: Standard
+created: 03/15/2021
+---
+
+# Summary
+
+This DIP is an extension of the Off-Chain protocol for enabling eCommerce P2M payments (Peer To Merchant) of the following types:
+* Charge
+* Auth / Capture
+
+This DIP defines the type of eCommerce P2M payments, corresponding requests and the payment structures. 
+ 
+This DIP extends [DIP-1](https://dip.diem.com/dip-1/) which specifies P2P payment flows (Peer To Peer).
+
+This DIP replaces the Auth/Capture section in [DIP-8](https://dip.diem.com/dip-8/#authcapture).
```

Comment:
Maybe we could revisit this, it seems rather jarring to have both fo these DIPs. Ideally this DIP would be self-contained.

Reply:
Maybe it wasn't clear, the section of auth/capture in DIP-8 will be removed.
DIP-8 will cover only funds pre-approval.
DIP-158 is self-contained.